packages/core/test/amqp/helpers/actions/success-set-header.js

```diff
@@ -0,0 +1,9 @@
+const { ActionTransport } = require('./../../../../src');
+
+function SuccessSetHeaderAction(serviceRequest) {
+  serviceRequest.setReplyHeader('x-your-response-header', 'header value');
```

Comment:
Why not use the simplest reply headers declaration ?
`serviceRequest.replyHeaders = { 'x-your-response-header' : 'header value' }`